src/single_simulator.cpp

```diff
@@ -27,12 +27,7 @@ namespace Nos3
 int
 main(int argc, char *argv[])
 {
-    if (argc != 2) {
-        std:: cout << "FATAL ERROR:  Expected a single argument, the name of a simulator (from the config file) to start" << std::endl;
-        return 1;
-    }
-
-    std::string simulator_name(argv[1]);
+    std::string simulator_name(argv[argc-1]); // assume that the last command line argument is the sim name to run
```

Comment:
Is it possible that argc could be passed in as zero?

Reply:
https://stackoverflow.com/questions/49817316/can-argc-be-zero-on-a-posix-system#:~:text=Yes%2C%20argv%5B0%5D%20can,was%20executed%20without%20any%20arguments.

So yes, it can... I'll work to protect the program from that possibility.